### PR TITLE
RT-1.3: BGP Route Propagation (KNE/OTG) - Adding delay of 5 seconds

### PR DESCRIPTION
**Pull request by stthummuri**
to come up on virtual environment

### CODE REVIEW

**Comment by stthummuri** [comment]
Added a delay of 5 seconds since it would take that time for bgp neighbors to come up on virtual environment

File: feature/bgp/addpath/otg_tests/route_propagation_test/route_propagation_test.go [hunk 1]
```diff
@@ -202,6 +202,7 @@ func (d *dutData) Configure(t *testing.T, dut *ondatra.DUTDevice) {
 
 func (d *dutData) AwaitBGPEstablished(t *testing.T, dut *ondatra.DUTDevice) {
 	for neighbor := range d.bgpOC.Neighbor {
+		time.Sleep(5 * time.Second)
```
Comment (greg-dennis) [inline feature/bgp/addpath/otg_tests/route_propagation_test/route_propagation_test.go:205]: I would add 5 seconds to the Await() call instead

**Review by greg-dennis — CHANGES_REQUESTED** [review]

**Comment by greg-dennis** [comment]
@stthummuri, are you going to continue this PR and resolve the rebase check?

**Comment by stthummuri** [comment]
> Details

@greg-dennis, yes I will use this PR only and will resolve the rebase check tomorrow

**Comment by stthummuri** [comment]
> @stthummuri, are you going to continue this PR and resolve the rebase check?

Resolved the rebase check. Can you please review and merge

**Review by greg-dennis — APPROVED** [review]
Re-approve

**Pull request merged**